recipes/stlab/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,25 @@
+// Async example from https://stlab.cc/libraries/concurrency/
+
+#include <iostream>
+#include <thread>
+
+#include <stlab/concurrency/default_executor.hpp>
+#include <stlab/concurrency/future.hpp>
+
+using namespace std;
+using namespace stlab;
+
+int main() {
+    auto f = async(default_executor, [] { return 42; });
+
+    // Waiting just for illustration purpose
+    while (!f.get_try()) { this_thread::sleep_for(chrono::milliseconds(1)); }
+
+    cout << "The answer is " << *f.get_try() << "\n";
```

Comment:
You can test the result of `f.get_try()` and return a non-zero integer when it fails.

Reply:
I don't want to change the example from the documentation.

Reply:
Ok. Checking whether the code runs fine should be enough.